app/driver.f90

```diff
@@ -116,12 +118,30 @@ subroutine run_main(config, error)
       call wrap_to_central_cell(mol%xyz, mol%lattice, mol%periodic)
    end if
 
+   ga = config%ga
+   gc = config%gc
    if (config%mbdscale) s9 = config%inp%s9
    if (config%rational) then
       if (config%has_param) then
          param = config%inp
       else
-         call get_rational_damping(config%method, param, s9)
+         is = index(config%method, '/')
+         if (is == 0) is = len_trim(config%method) + 1
+         functional = lowercase(config%method(:is-1))
```

Comment:
Can we get the functional ID instead and compare this instead of the string?

Reply:
We could, but then we have to take care to never change the order in the ID enumerator.

Reply:
That would be fine just need to be internally consistent.